tests/E2E/test/selectors/BO/catalogpage/category_submenu.js

```diff
@@ -1,44 +1,44 @@
-    delete_action_group_button: '//*[@id="form-category"]//a[contains(@onclick, "Delete selected")]',
-    category_number_span: '//*[@id="form-category"]//span[@class="badge"]',
-    category_description: '//*[@id="category_form"]//div[@class="mce-tinymce mce-container mce-panel"]',
-    description: '//*[@id="table-category"]//td[4]',
-    delete_tag_button: '//div[@class="tagify-container"]//span[%POS]/a',
-    mode_link_disable_radio: '//*[@id="deleteMode_linkanddisable"]',
-    mode_link_radio: '//*[@id="deleteMode_link"]',
-    mode_delete_radio: '//*[@id="deleteMode_delete"]',
+    title: '//*[@id="category_meta_title_1"]',
+    meta_description: '//*[@id="category_meta_description_1"]',
+    keyswords: '//*[@id="category_meta_keyword_1-tokenfield"]',
+    simplify_URL_input: '//*[@id="category_link_rewrite_1"]',
+    save_button: '//form[@name="category"]//*[contains(@class,"card-footer")]/button',
+    reset_button: '//*[@id="category_grid"]//thead//button[@type="reset"]',
+    search_input: '//input[@id="category_name"]',
+    search_button: '//button[@name="category[actions][search]"]',
+    search_result: '//*[@id="category_grid"]//td[3]',
+    update_button: '//*[@id="category_grid"]//tbody//td[7]//a[@data-original-title="Edit"]',
+    action_button: '//*[@id="category_grid"]//td[7]//a[@data-toggle="dropdown"]',
+    delete_button: '//*[@id="category_grid"]//td[7]//a[contains(@class,"delete-category")]',
+    view_button: '//*[@id="category_grid"]//td[7]//a[@data-original-title="View"]',
+    image_link: '//form[@name="category"]//label[contains(text(),"Category cover image")]/..//figure/img',
+    thumb_link: '//form[@name="category"]//label[contains(text(),"Category thumbnail")]/..//figure/img',
+    second_delete_button: '//div[@class="modal-footer"]//button[contains(@class,"submit-delete-categories")]',
+    select_category: '//*[@id="category_grid"]//tbody/tr/td//input[@type="checkbox"]/../..',
+    action_group_button: '//*[@id="category_grid"]//button[contains(@class, "dropdown-toggle")]',
+    delete_action_group_button: '//*[@id="category_grid_bulk_action_delete_selection"]',
+    category_number_span: '//*[@id="category_grid_panel"]//h3',
+    category_description: '//textarea[@id="category_description_1"]',
+    description: '//*[@id="category_grid"]//td[4]',
+    delete_tag_button: '//div[@class="token"][%POS]/a',
+    mode_link_disable_radio: '//*[@id="delete_categories_delete_mode_0"]/..',
+    mode_link_radio: '//*[@id="delete_categories_delete_mode_1"]/..',
+    mode_delete_radio: '//*[@id="delete_categories_delete_mode_2"]/..',
     expand_all_button: '#expand-all-categories-tree',
     parent_category: '(//*[@id="categories-tree"]//span[contains(.,"%NAME")]//input)[1]',
-    search_no_results: '//*[@id="table-category"]//td[@class="list-empty"]',
+    search_no_results: '//*[@id="category_grid"]//tbody/tr/td//p[2]',
```

Comment:
This selector seems not resilient to future changes on the theme. Can are you trying to target here? We should try to make the selection easier.

Reply:
For this selector, I can't do better.
I suggest to add an id for migration project.

Reply:
I guess it's the same as in https://github.com/PrestaShop/PrestaShop/pull/13570#discussion_r278834899? If yes, let's wait for the other PR to be merged and you will be able to improve this selector.

Reply:
Ok for me